resources/usa_bars/sv_bars.lua

```diff
@@ -23,6 +23,7 @@ local ITEMS = {
       ["Voodoo"] = {name = "Voodoo", price = 60, type = "alcohol", substance = 15.0, quantity = 1, legality = "legal", weight = 10, strength = 0.3, caption = "Ketel One Vodka and Chambord, shaken ice cold with fresh blueberries, smoked jalapenos and lemon juice."},
       ["Mi Casa Es Su Casa"] = {name = "Mi Casa Es Su Casa", price = 60, type = "alcohol", substance = 15.0, quantity = 1, legality = "legal", weight = 10, strength = 0.28, caption = "A premium blend of Casamigos Blanco Tequila and Cointreau, shaken ice cold with fresh lime juice, agave, a hint of orange and olive juice."},
       ["Back Porch Strawberry Lemonade"] = {name = "Back Porch Strawberry Lemonade", price = 60, type = "alcohol", substance = 15.0, quantity = 1, legality = "legal", weight = 10, strength = 0.28, caption = "Skyy Infusions Wild Strawberry, Triple Sec, Strawberry Lemonade and a splash of soda. Served over ice in our 22 oz mug with fresh strawberries."},
+      ["Pepsi"] = {name = "Pepsi", price = 75, type = "drink", substance = 38.0, quantity = 1, legality = "legal", weight = 5, strength = 0.38, caption = "A refreshing carbonated drink with a citrusy flavor burst."},
```

Comment:
```suggestion
      ["Pepsi"] = {name = "Pepsi", price = 75, type = "drink", substance = 38.0, quantity = 1, legality = "legal", weight = 5, caption = "A refreshing carbonated drink with a citrusy flavor burst."},
```

strength is for alcoholic drinks p sure lol